src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -101,7 +101,7 @@ def set_storage_account_properties(
 def upload_blob( # pylint: disable=too-many-locals
         client, container_name, blob_name, file_path, blob_type=None,
         content_settings=None, metadata=None, validate_content=False, maxsize_condition=None,
-        max_connections=2, max_retries=5, retry_wait=1, lease_id=None, if_modified_since=None,
```

Comment:
Yes this was a change in the Storage data plane SDK - https://github.com/Azure/azure-storage-python/blob/master/BreakingChanges.md#version-0330.